src/app/routes/onDemandRadio/getInitialData/index.js

```diff
@@ -35,6 +35,15 @@ const getEpisodeAvailableUntil = path([
   'availableUntil',
 ]);
 const getReleaseDateTimeStamp = path(['metadata', 'releaseDateTimeStamp']);
+const getPromoBrandTitle = path(['promo', 'brand', 'title']);
+const getDurationISO8601 = path([
+  'promo',
+  'media',
+  'versions',
+  0,
+  'durationISO8601',
+]);
+const getImageUrl = path(['promo', 'media', 'imageUrl']);
```

Comment:
As mentioned in https://github.com/bbc/simorgh/pull/6461#discussion_r421626443 we might need to do some form of conversion on the image url